### PR TITLE
Add latency to the event timestamp tooltip.

### LINKED ISSUE #3283: Improve display for events that are received far after the occurrence timestamp.
A couple ideas on how to improve this:

- show some sort of indicator (e.g. :warning:) next to the event timestamp when the difference between the difference between the occurred and received timestamps exceed some threshold (60 minutes?)
- show the difference between the received and occurred timestamps as part of the tooltip when hovering over the event timestamp so that you don't have to manually do math to understand how delayed the event was

### PR DESCRIPTION
**Pull request by tkaemming**
screenshot 2016-05-25 12 16 11

Fixes GH-3283.

@getsentry/ui @getsentry/javascript

### CODE REVIEW

**Comment by mattrobenolt** [comment]
I'd like to have a quantifier here. Like, "seconds", "minutes", "hours", etc.

What about even if it was just the fuzzy "1 hour ago" style? I think that's reasonable here and doesn't require it to be an exact value. I think that'd be easier to grok at a glance.

**Comment by tkaemming** [comment]
screenshot 2016-05-25 13 24 55
screenshot 2016-05-25 13 25 03

**Comment by mattrobenolt** [comment]
That's significantly more readable. :+1:

What about timestamps in the future? Or is this just an absolute difference?

**Comment by tkaemming** [comment]
I switched this to use the absolute value between the two timestamps, but didn't include anything that shows if the difference is positive or negative because I couldn't think of any wording that makes sense. A negative latency value seems confusing (to me, at least) as does "Latency: 37 hours in the future". I'm open to changing this if anybody has ideas. "Offset" could make sense, but I'm not sure that's a very clear way to articulate what this represents.

**Comment by mattrobenolt** [comment]
The future case is significantly less common since we only allow it to be drifted 1 minute into the future. Beyond that, we'll show an error message that it's too far in the future.

So I think this is fine as-is.

Side note: I thought moment could do this humanized formatting itself, complete with localization built in. I know it'll support it with `fromNow()`

@benvinegar

**Comment by tkaemming** [comment]
> Side note: I thought moment could do this humanized formatting itself, complete with localization built in. I know it'll support it with fromNow()

The `humanize` method skews the representation in a way that I find confusing: https://github.com/moment/moment/blob/develop/src/test/moment/duration.js#L331-L365

**Comment by tkaemming** [comment]
Added a warning icon if the latency is over 30 minutes:

screenshot 2016-05-25 14 31 07

File: src/sentry/static/sentry/less/group-detail.less [hunk 1]
```diff
@@ -711,6 +711,12 @@
     color: @gray-dark;
   }
 
+  .icon-alert {
```
Comment (tkaemming) [inline src/sentry/static/sentry/less/group-detail.less]: This isn't a very specific selector, but from what I can tell there are a lot of places where we keep the selectors pretty generic. If my impression here is wrong, I'm happy to change it to something more specific (e.g. `.latency.icon-alert`) or something.

**Pull request merged**